analytics/src/analytics/cli.py

```diff
@@ -61,18 +61,14 @@
 app.add_typer(etl_app, name="etl", help="Transform and load local file")
 
 
-def init() -> None:
-    """Shared init function for all scripts."""
-    # Setup logging
-    init_logging(__package__)
-    init_app(logging.root)
+# Setup logging
+init_logging(__package__)
+init_app(logging.root)
```

Comment:
This is potentially going to make local dev / testing more difficult if you always enable logging as soon as the module loads.

What is the context for this change / where do you want logging that you currently don't have it?